covidfaq/bert_fr_model/config.yaml

```diff
@@ -4,8 +4,15 @@ dev_files:
 test_file: '../../covidfaq_data/crowdsource/20200416_quebec_faq_fr_cleaned_collection1_2_3_newformat/validation.json'
 keep_ood: false
 
-ckpt_to_resume: covidfaq/bert_fr_model/output/bert_model.ckpt 
-outlier_model_pickle: covidfaq/bert_fr_model/output/sklearn_outlier_model.pkl
+ckpt_to_resume: covidfaq/bert_fr_model/output/bert_model.ckpt
+
+outlier:
+    ood_filename: 'covidfaq/bert_en_model/fr_ood_model.pkl'
+    model_name: 'local_outlier_factor'
+    training_data_files:
+        - covidfaq/data/covidfaq_data/crowdsource/20200416_quebec_faq_fr_cleaned_collection1_2_3_newformat/train.json
+        - covidfaq/data/covidfaq_data/crowdsource/20200416_quebec_faq_fr_cleaned_collection1_2_3_newformat/validation.json
+        - covidfaq/data/covidfaq_data/crowdsource/20200522_quebec_faq_fr_cleaned_collection4/test.json
```

Comment:
@jerpint It seems like these are not the right filenames. Could you provide the correct ones please?